roles/azimuth_caas_operator/defaults/main.yml

```diff
@@ -131,6 +162,7 @@ azimuth_caas_stackhpc_slurm_appliance_template:
   envVars:
     # Normally set through environment's activate script:
     ANSIBLE_INVENTORY: environments/common/inventory,environments/.caas/inventory # NB: Relative to runner project dir
+  access_controls: "{{ azimuth_caas_stackhpc_slurm_appliance_tenancy_access_control_annotations }}"
```

Comment:
I'd prefer if we introduced another level here, e.g.

```yaml
azimuth_caas_stackhpc_slurm_appliance_template:
  annotations: "{{ azimuth_caas_stackhpc_slurm_appliance_tenancy_access_control_annotations }}"
  spec:
    gitUrl: "{{ azimuth_caas_stackhpc_slurm_appliance_git_url }}"
    ...
```

I think it makes it clearer. Using the key `annotations` allows us to potentially add other annotations in future using `combine` without changing the `tasks` code.

Same for other places.